src/serac/infrastructure/tests/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 #
 # SPDX-License-Identifier: (BSD-3-Clause) 
 
-set(test_dependencies gtest serac_physics serac_state)
+set(test_dependencies gtest serac_physics)
```

Comment:
This goes back to @samuelpmishLLNL 's comments, and outside the scope of this PR, about our unit tests being oddly over dependent.

This directory should not be dependent on `serac_physics` but is.